resources/locale/en.yml

```
@@ -21,7 +21,8 @@ fof-discussion-language:
       show_flag_label: Show country flags with language names
       composer_default_label: Pre-select current locale in the discussion composer
       locale_sort_label: Use detected language for default discussion filter
-      show_any_lang_opt_label: Show the option to select "Any Language".
+      show_any_lang_opt_label: Show the option to select "Any Language"
+      tags_page_discussion_locale_label: Only show discussions from the user's locale on the Tags page
```

```suggestion
      tags_page_discussion_locale_label: Only show discussions from the user's current locale on the Tags page
```

Let's give a clear idea that by switching language, this will also switch the locale used on the Tags page discussions